server/main.py

```diff
@@ -561,6 +669,7 @@ def api_users():
 @app.errorhandler(500)
```

Comment:
TBH this entire handler should probably be deleted, it's only here because it's from a sample flask app.

Reply:
done